### ISSUE TITLE
Button 1, 2, 4 and 5 does not work on Razer Naga Trinity

### ISSUE THREAD

**Issue by GeneralUllsokk** [lostallmymoney/Razer_Key_Mapper_Linux#3]
Hey!

I have used your program for a little while now and found it quite useful. When I first installed it all my side buttons worked but after a while button 1, 2, 4 and 5 stopped working. They do not respond when I assign a key to them via the "keyMap.txt" file. I have tested the buttons on a Windows machine and have concluded that they are not broken. 

I am running Manjaro 

`Operating System: Manjaro Linux
            Kernel: Linux 5.2.21-1-MANJARO
      Architecture: x86-64
`

This is my output from xinput -list 

⎡ Virtual core pointer                          id=2    [master pointer  (3)]
⎜   ↳ Razer Razer Naga Trinity                  id=14   [slave  pointer  (2)]
⎜   ↳ Razer Razer Naga Trinity Consumer Control id=16   [slave  pointer  (2)]

⎣ Virtual core keyboard                         id=3    [master keyboard (2)]
    ↳ Razer Razer Naga Trinity Keyboard         id=15   [slave  keyboard (3)]
    ↳ Razer Razer Naga Trinity System Control   id=17   [slave  keyboard (3)]
    ↳ Razer Razer Naga Trinity                  id=18   [slave  keyboard (3)]
    ↳ Razer Razer Naga Trinity Consumer Control id=20   [slave  keyboard (3)]

Any help would be appreciated!

**Comment by lostallmymoney** [comment]
So as I understand there was no problem when you 1st used the program ? And are these buttons the interchangeable ones ? I bet the problem is in xinput but I'm in a very bad place to help you right now. I'll get back to you shortly.

**Comment by GeneralUllsokk** [comment]
I have Windows installed on another SSD (dual boot) and use Razer Synapse there. When I do NOT run your program on Manjaro and press the side button "1" it prints out "q". This is directly what I configured the mouse on Razer Synapse on Windows. I thought the configuration was only in the software, but it seems that the config from Razer Synapse (on Windows) is still in effect when I have booted Manjaro.

When I do start your program all buttons except 1, 2, 4 and 5 works.

And to answer your questions: 
All buttons worked before I configured the same mouse on Razer Synapse on Windows. After I did that button 1, 2, 4 and 5 stopped responding with your program on Manjaro. Yes, I have the buttons which are interchangeable.

It looks to me that I need to do a hard reset of my mouse of some sort, but I am not sure how to do this on Manjaro. 

Thanks for helping me with this

**Comment by lostallmymoney** [comment]
Well this is beyond my knowledge looks like the mouse has some memory inside and acts as a keyboard. Maybe you can find that keyboard with xinput and try to assign it in the xinput script. I think there's a way to see wich keyboard it is. But maybe if you change your config on windows you would have to change the config on linux every time. I'd need a mouse to test

**Comment by lostallmymoney** [comment]
So to see wich keyboard it is i suggest using command line and xinput to disable and re enable every keyboard you see on your mouse until you find the one for the buttons.

**Comment by GeneralUllsokk** [comment]
I fixed the issue by resetting the buttons on a Windows PC with Synapse. Now all my buttons work perfectly with your software. I have seen different posts online with different buttons not working. A solution for this may be to reset all buttons on Synapse where in my case the buttons which did not work, was the ones that I resetted. I had them mapped to a keyboard button through Synapse, which made them not working with your software.
